### DISCUSSION TITLE
Where is everything installed on the local unraid server?

### DISCUSSION THREAD

**Discussion by ochompsky** [RobertDWhite/WhiteMatterTech#17] [General]
where is the qemu image located?

**Comment by RobertDWhite** [comment]
Hi @ochompsky . Which article are you referring to specifically? 

Thanks!

**Reply by ochompsky** [reply]
hi, so im following this tutorial - https://whitematter.tech/posts/run-dockerized-macos-on-unraid/

during install there's a 240gb image on the installer, im curious, where is all of that data stored within the context of unraid?  basically the osx installer.  with a traditional docker container, i have a volume in appdata but im not sure how it works with this.

**Reply by Spiceapps** [reply]
i have the same question - is it all happening inside the docker.img file in unraid? should i make sure it is large enough to contain the installation media, etc.
and thanks for your tutorial, it is very clear.

**Reply by RobertDWhite** [reply]
Hi all,

Sorry I missed the responses here. I really need to get my GitHub notifications set better.

Docker stores images and containers on Unraid in the docker.img. The size of this file should be large enough to hold all the Docker images, containers, and any data stored within the containers. You can increase the size of the .img in Unraid settings. I hope this helps!
